src/file.ts

```diff
@@ -2180,6 +2214,8 @@ class File extends ServiceObject<File> {
    * @param {object} config Configuration object.
    * @param {string} config.action "read" (HTTP: GET), "write" (HTTP: PUT), or
    *     "delete" (HTTP: DELETE), "resumable" (HTTP: POST).
+   * @param {string} [config.version='v2'] The signing version to use, either
+   *     'v2' or 'v4.
```

Comment:
Add a warning when enabling v2 that the default will change in the future:
 
> You have generated a signed URL using the default v2 signing implementation. In the future, this will default to v4. You may experience breaking changes if you use longer than 7 day expiration times with v4. To opt-in to the behavior specify `config.version='v4'`